src/instruments/src/EFB/ChartsApi/ChartFox.tsx

```diff
@@ -24,7 +24,7 @@ export default class ChartFoxClient {
         if (ChartFoxClient.sufficientEnv()) {
             if (icao.length === 4) {
                 const chartJsonResp = await fetch(`https://chartfox.org/api/charts/grouped/${icao}?token=${ChartFoxClient.token}`, { method: 'POST' }).catch(() => {
-                    console.log('Token Authentication Failed. #CF101');
+                    throw new Error('Token Authentication Failed. #CF101');
```

Comment:
Are we sure we want to be throwing an error rather than `console.error()` ? Throwing an error stops execution. However someone more experienced in error handling can say what's the better choice.

Reply:
Preferably, these would not be console.logs and actual thrown Errors that could be handled when called. I haven't gotten back to this PR but I will need to implement the catching of the exception.